src/components/pages/leaderboard/hero/hero.jsx

```diff
@@ -1,41 +1,81 @@
+import Image from 'next/future/image';
 import Link from 'next/link';
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import DiscordIcon from './images/discord.inline.svg';
-import TwitterIcon from './images/twitter.inline.svg';
+import Socials from 'components/shared/socials';
 
-const title = '>>Leaderboard';
+import bgLeftGlitch from './images/bg-left-glitch.png';
+import bgRightGlitch from './images/bg-right-glitch.png';
+import bgTitleGlitch from './images/bg-title-glitch.png';
+
+const title = '>_Leaderboard';
+const leadersHeader = ['Place', 'Name', 'Score'];
 
 const Hero = ({ teams }) => (
   <section className="safe-paddings relative min-h-[600px]">
-    <div className="container relative z-10 flex h-full flex-col items-center justify-center">
+    <Image
+      className="absolute top-[70px] left-0 -z-20 md:top-10 sm:hidden"
+      src={bgLeftGlitch}
+      width={464}
+      height={78}
+      loading="eager"
+      alt="Left Glitch image"
+      priority
+      aria-hidden
+    />
+    <Image
+      className="absolute left-0 right-0 top-0 -z-10 lg:hidden"
+      src={bgTitleGlitch}
+      width={1920}
+      height={219}
+      loading="eager"
+      alt="Center Glitch image"
+      priority
+      aria-hidden
+    />
+    <Image
+      className="absolute right-0 top-9 -z-20 md:top-12 sm:hidden"
+      src={bgRightGlitch}
+      width={474}
+      height={105}
+      loading="eager"
+      alt="Right Glitch image"
+      priority
+      aria-hidden
+    />
```

Comment:
Let's put these images after the main blocks at the end because they are decorative elements - this will improve the readability of the code since we will initially see the main information